netpyne/network/pop.py

```diff
@@ -290,7 +290,12 @@ def createCellsDensity(self):
         self.rand.uniform(0, 1)
         vec = h.Vector(self.tags['numCells'] * 3)
         vec.setrand(self.rand)
-        randLocs = np.array(vec).reshape(self.tags['numCells'], 3)  # create random x,y,z locations
+        try:
+            randLocs = np.array(vec).reshape(self.tags['numCells'], 3)  # create random x,y,z locations
+        except Exception as e:
+                if 'numCells' in self.tags and self.tags['numCells'] == 0:
+                    print("Unable to create network, please validate that cell population > 0")
```

Comment:
I'd propose to also include pop name to error message.. Sorry for being pedantic:)
How about e.g. `print(f"Unable to create network, please validate that '{self.tags['pop']}' population size is > 0 ")`